src/components/Form.js

```diff
@@ -19,11 +27,19 @@ const Form = () => {
           onChange={onTitleChanged}
           value={title}
         />
-        <select name="author" value={author} onChange={onAuthorChanged}>
+        <input
+          type="text"
+          name="author"
+          id="author"
+          placeholder="Book Author"
+          onChange={onAuthorChanged}
+          value={author}
+        />
+        {/* <select name="author" value={author} onChange={onAuthorChanged}>
           <option value="Mary">Mary</option>
           <option value="Grey">Grey</option>
-        </select>
-        <button type="submit">Add Book</button>
+        </select> */}
```

Comment:
- [OPTIONAL] It is professional to remove unused code from your codebase. It also helps the scalability of your app.